### PR TITLE
Update hammer_cli and hammer_cli_foreman to 0.10.0

### PR DESCRIPTION
**Pull request by tstrachota**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Scratchbuilds:
- [hammer_cli](http://koji.katello.org/koji/taskinfo?taskID=617644)
- [hammer_cli_foreman](http://koji.katello.org/koji/taskinfo?taskID=617661)

### CODE REVIEW

File: rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 1]
```diff
@@ -27,8 +27,7 @@ Requires: %{?scl_prefix}rubygem(awesome_print)
 Requires: %{?scl_prefix}rubygem(highline)
 Requires: %{?scl_prefix}rubygem(fast_gettext)
 Requires: %{?scl_prefix}rubygem(locale) >= 2.0.6
-Requires: %{?scl_prefix}rubygem(apipie-bindings) >= 0.0.14
-Requires: %{?scl_prefix}rubygem(apipie-bindings) < 0.1.0
```
Comment (ehelms) [inline rubygem-hammer_cli/rubygem-hammer_cli.spec]: Why remove this limitation here?

Comment (mmoll) [inline rubygem-hammer_cli/rubygem-hammer_cli.spec]: it's reflecting the gemspec

Comment (ehelms) [inline rubygem-hammer_cli/rubygem-hammer_cli.spec]: Thanks for clarifying @mmoll

**Comment by ehelms** [comment]
Looks like rubygem-hammer_cli_foreman/hammer_cli_foreman-0.10.0.gem was added as a binary file and not with git annex.

**Comment by tstrachota** [comment]
Oops, fixed @ehelms

**Comment by ehelms** [comment]
Thanks @tstrachota !

**Pull request merged**